health_check_test.go

```diff
@@ -88,23 +90,16 @@ var _ = Describe("HealthCheck", func() {
 			queueManager.Close()
 		})
 
-		It("should return a status struct showing the status of RabbitMQ and Redis", func() {
+		It("has a successful overall system status", func() {
```

Comment:
I think the term "success" is ambiguous. Was the check successful because it returned or because it returned OK? We have adopted the terms "ok", "warning", and "critical" so we should use those when we describe the behaviour in our test too.

Maybe we could say that it "reports that the health of the system is OK".